apache/superset
# fix: Unable to download the Dashboard as image in case there's an image added through Markdown

### SUMMARY

Users can add Markdown to their Dashboards, which allow to include images using `<img src="$link">`. However, if the Dashboard has an image added this way, the Download as image functionality will fail. 

The problem is that, depending on where do we source the image from, we could run into CORS issues.
That, however, show not make the image generation fail.

The library that we use, `dom-to-image`, is pretty much deprecated and hasn't received updates for a long while.

This PR replaces the library for a fork that's been rewritten and well maintained: `dom-to-image-more`.
The library does not fail, and instead shows a placeholder if it can't fetch the image.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Before:

https://user-images.githubusercontent.com/17252075/173392024-12b0cfb4-a5a6-4067-9c8f-d3cb680bd219.mov

After:

https://user-images.githubusercontent.com/17252075/173392092-46384421-d0cd-4331-a5b1-c156dd6b99c2.mov

### TESTING INSTRUCTIONS
1. Access your Workspace.
2. Create a new Dashboard.
3. Add a Markdown element to it.
4. Add below code to your Markdown:
```
<img src="https://repository-images.githubusercontent.com/39464018/58649580-eca4-11ea-844d-c2ddca24b226">
 ```
5. Save your changes.
6. Click on the three ellipses > Download as image.

Ensure the dashboard image is generated.

### ADDITIONAL INFORMATION
